contrib/gocql/gocql/gocql.go

```diff
@@ -111,17 +111,16 @@ func (tq *Query) newChildSpan(ctx context.Context) ddtrace.Span {
 }
 
 func (tq *Query) finishSpan(span ddtrace.Span, err error) {
-	if err == gocql.ErrNotFound {
-		// ErrNotFound is technically an error from gocql point of view.
+	if err != nil && tq.params.config.errCheck != nil && tq.params.config.errCheck(err) == false {
```

Comment:
```suggestion
	if err != nil && tq.params.config.errCheck != nil && !tq.params.config.errCheck(err) {
```
Perhaps we could have a function in `*config` which checks both these things. Seems a bit boilerplatey, something like:
```go
func (c *config) ignoreError(err error) bool {
    return c.errCheck != nil && !c.errCheck(err)
}
```